src/App.tsx

```diff
@@ -15,71 +16,96 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedProduct: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedProduct: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  render() {
+    return (
+      <main className="section container">
+        <h1 className="title is-flex is-align-items-center">
+          {
+            this.state.selectedProduct
+              ? `${this.state.selectedProduct} is selected`
+              : 'No goods selected'
+          }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
let's google eslint rule end fix it, instead of disabling
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/control-has-associated-label.md